dev/release/02-source.sh

```diff
@@ -37,11 +37,10 @@ fi
 version=$1
 rc=$2
 
-tag=apache-arrow-${version}
+tag=apache-arrow-${version}-rc${rc}
```

Comment:
This change is OK but we want to use `apache-arrow-X.Y.Z.tar.gz` not `apache-arrow-X.Y.Z-rcN.tar.gz` for archive name.

Reply:
Ah ok. I'll update the `tarball` name to not include `-rcN`.